src/GraphRequest.ts

```diff
@@ -631,12 +629,8 @@ export class GraphRequest {
 		const options: FetchOptions = {
 			method: RequestMethod.GET,
 		};
-		try {
-			const response = await this.send(url, options, callback);
-			return response;
-		} catch (error) {
-			throw error;
-		}
+		const response = await this.send(url, options, callback);
```

Comment:
return the response directly? (x5/6)

Reply:
Vincent's comment is applicable to lines 658, 689, 708, 736, 764, and 785

Reply:
Done